packages/ag-charts-community/src/chart/series/cartesian/markerUtil.ts

```diff
@@ -134,11 +136,37 @@ export function markerScaleInAnimation<T>(
         animationManager,
         markerSelections,
         { scalingX: 0, scalingY: 0 },
-        { scalingX: 1, scalingY: 1 },
-        { duration: animationManager.defaultDuration }
+        { scalingX: 1, scalingY: 1 }
     );
+    markerSelections.forEach((s) => s.cleanup());
+}
+
+export function markerSwipeScaleInAnimation<T extends CartesianSeriesNodeDatum>(
+    { id }: { id: string },
+    animationManager: AnimationManager,
+    markerSelections: Selection<Node, T>[],
+    seriesWidth: number
+) {
+    const fromFn = (_: Node, datum: T) => {
+        const x = datum.midPoint?.x ?? seriesWidth;
+        const delayRatio = easing.easeInOut(x / seriesWidth) - 0.1;
+        const delay = Math.max(Math.min(delayRatio, 1), 0);
+        return { scalingX: 0, scalingY: 0, animationDelay: delay, animationDuration: 0.2 };
```

Comment:
Should `0.2` be some constant?

Reply:
There are plans to make this a user option on animations, but it remains to be seen how that is done. Previously this was just a number of `200` as milliseconds in various places. But yeah, probably.

Reply:
Added a more general constant for these 'fast' ad-hoc transitions.